src/Symfony/Bridge/Twig/Tests/Extension/FormExtensionBootstrap3HorizontalLayoutTest.php

```diff
@@ -66,42 +66,42 @@ protected function renderLabel(FormView $view, $label = null, array $vars = [])
             $vars += ['label' => $label];
         }
 
-        return (string) $this->renderer->searchAndRenderBlock($view, 'label', $vars);
+        return $this->renderer->searchAndRenderBlock($view, 'label', $vars);
```

Comment:
`FormRendererInterface::searchAndRenderBlock()` retturns a string (same for the others)